lib/sentry.ex

```diff
@@ -121,7 +121,7 @@ defmodule Sentry do
     end
 
     validate_json_config!()
-    validate_log_level_config!()
+    Config.validate_log_level!()
```

Comment:
If we move this to the `Sentry.Config` module, we don't need to expose the "permitted log level values" from there.